scripts/examples/nrfconnect_example.sh

```diff
@@ -16,6 +16,9 @@
 #    limitations under the License.
 #
 
+# Run bootstrap to set up e.g. Pigweed correctly
+source "$(dirname "$0")/../../scripts/bootstrap.sh"
```

Comment:
This takes 3 minutes on CI, I wonder if it was possible to pre-configure our Docker images to make Pigweed initialize itself faster. But of course this is unrelated to your change. Just thinking :)

Reply:
we've asked pigweed to look into working in a docker environment